Sources/Rendering/OpenGL/RenderWindow/index.js

```diff
@@ -311,6 +315,21 @@ function vtkOpenGLRenderWindow(publicAPI, model) {
         model.xrReferenceSpace = refSpace;
       });
 
+      const renderer = model.renderable.getRenderers()[0];
+      const offset = renderer.getActiveCamera().getPhysicalTranslation();
+      let scale = renderer.getActiveCamera().getPhysicalScale();
+      if (model.xrSessionIsAR) {
+        // Default setting for AR is slightly smaller / more distant
+        scale *= 3;
+        offset[2] *= -5;
+        renderer.getActiveCamera().setPhysicalScale(scale);
+        renderer.getActiveCamera().setPhysicalTranslation(offset);
+      } else {
+        // Move VR scene a bit forward so that we don't start in the middle of objects
+        offset[2] *= -3;
+        renderer.getActiveCamera().setPhysicalTranslation(offset);
+      }
```

Comment:
These settings in `enterXR` adjust the world-to-physical matrix we originally get from `renderer.resetCamera()` a bit so that the scene is a reasonable size for VR vs AR. The values here are arbitrary and based on testing for what makes the cone scene "comfortable", I'd welcome suggestions on how we can refine this.

Reply:
After more testing, these adjustments don't completely solve the positioning issue. Half the time the offset will move the scene in the wrong direction so that the scene is behind the viewer. Scaling/positioning the scene for VR or AR experiences is still a consideration that should be pursued, but perhaps is best left out of these initial changes.